app/assets/javascripts/transactions.js

```diff
@@ -51,26 +46,41 @@ $(function () {
     $('#new_transaction').show();
   });
 
+  $(document).on('change', '#transfer_amount', function(e) {
+    prep_rate_and_hints(exchange_rate = false, hints = true)
```

Comment:
Made it to clarify what we are expected to be shown by this function (don't show rate. but show hints)